.github/workflows/cli-core-test.yml

```diff
@@ -0,0 +1,55 @@
+name: Cli Core Tests
```

Comment:
Minor: Can you format the PR title to the required format? It throws up everytime I run something on Librarian. You'll have to mention if it's a fix or feature or whatever is suitable.